src/main/java/com/cloudant/http/HttpConnection.java

```diff
@@ -68,6 +67,23 @@
 public class HttpConnection {
 
     private static final Logger logger = Logger.getLogger(HttpConnection.class.getCanonicalName());
+
+    private static final String USER_AGENT;
+
+    static {
+        String ua = "java-cloudant-http/unknown";
```

Comment:
It's something we could perhaps sort out later, basically, in sync-android when sync is built on java-cloudant, we'll be able to do alot more, so the version stuff can be expanded to a few more methods etc to make it easier, but thats a long way down the line yet.